JumpscaleLibs/clients/minio/MinioClient.py

```diff
@@ -0,0 +1,114 @@
+"""
+Minio Healing Client
+"""
+
+from Jumpscale import j
+import requests
+
+JSConfigClient = j.baseclasses.object_config
+
+
+class MinioClient(JSConfigClient):
+    """
+    Minio healing client object
+    """
+
+    _SCHEMATEXT = """
+        @url = jumpscale.minio.client
+        name** = "" (S)
+        url** = "" (S)
+        """
+
+    def get_jobs(self):
+        resp = requests.get(f"{self.url}/jobs")
+        resp.raise_for_status()
+        jobs = resp.json()
+    
+        minio_jobs = []
+        for job_id, job in jobs.items():
+            minio_job = MinioJob(
+                self.url, job_id, job["location"], job["type"], job["running"]
```

Comment:
This doesn't return the job statistics. If you run a job as `background` job, the job instance from the server will have more data than just this.